pandas/conftest.py

```diff
@@ -1081,6 +1087,7 @@ def iris(datapath):
     timezone.utc,
     timezone(timedelta(hours=1)),
     timezone(timedelta(hours=-1), name="foo"),
+    ZoneInfo("US/Pacific"),
```

Comment:
`US/Pacific` is a deprecated zone. `America/Los_Angeles` is the preferred key ID.